test/cases/api.sh

```diff
@@ -99,6 +99,15 @@ BASEURLMAJORVERSION="http://$ADDRESS:$PORT/api/image-builder/v1"
 REQUEST_FILE="${WORKDIR}/request.json"
 ARCH=$(uname -m)
 
+RH_SUBSCRIPTION=",
```

Comment:
I'd actually prefer the duplication in each of the createReqFile*() functions rather than using a variable, makes it easier to read the exact request from the code.

Reply:
Updated. Thanks!